dali/kernels/slice/slice_kernel_test.h

```diff
@@ -242,6 +242,22 @@ struct ArgsGen_CompletelyOutOfBounds{
   }
 };
 
+template <typename OutputType, int Dims = 3>
+struct ArgsGen_SingleValuePad {
+  SliceArgs<OutputType, Dims> Get(const TensorShape<Dims>& input_shape) {
+    SliceArgs<OutputType, 3> args;
+    args.anchor[0] = -input_shape[0] / 2;
+    args.anchor[1] = -input_shape[1] / 2;
+    args.anchor[2] = 0;
+    args.shape[0] = 2 * input_shape[0];
+    args.shape[1] = 2 * input_shape[1];
+    args.shape[2] = input_shape[2];
+    args.fill_values = {128};
+    args.channel_dim = -1;
```

Comment:
From the description of the PR the bug was supposed to happen, when the channel dim is last, but there is no channel slicing. Doesn't this specify a no-channel variant with the -1?

Reply:
No, channel_dim=-1 means I don't care which dimension is channels. That is because we have a single pad value (128 in this case) so the channel dimension doesn't need special treatment